skimage/feature/texture.py

```diff
@@ -291,3 +293,130 @@ def local_binary_pattern(image, P, R, method='default'):
     image = np.ascontiguousarray(image, dtype=np.double)
     output = _local_binary_pattern(image, P, R, methods[method.lower()])
     return output
+
+def multiblock_local_binary_pattern(int_image, x, y, width, height):
+    """Multi-block local binary pattern.
+
+    MB-LBP is an extension of LBP that can be computed on many
+    scales in a constant time using integral image. It consists of
+    9 equal-sized rectangles. Sum of pixels' intensity values
```

Comment:
@stefanv 
@vighneshbirodkar 
Yes, you are right.
I think we can change it.
But I can see one problem with it:
this can later confuse users, because if they provide size that is not `% 3 == 0`
it will be fitted to this size without their will.

What do you think?